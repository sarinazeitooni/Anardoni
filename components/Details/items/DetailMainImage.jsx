import React from 'react';
import style from './scss/DetailsMainImage.module.scss';
const DetailMainImage = ({item}) => {
    return (
        <fragment>
            <div className={style.detailMainImageContainer}>
                <div className={style.darkerBack}></div>
                <div className={style.blurredImage}
                     style={{ backgroundImage: `url(
                      ${item.containerImage}
                     )` }}></div>
                <div className={style.detailMainImage}
                     style={{ backgroundImage: `url(
                      ${item.containerImage}
                     ) ` }}
                ></div>
            </div>
            <div className={style.caption}>
                {item.caption}
            </div>
        </fragment>
    )
};
export default DetailMainImage;