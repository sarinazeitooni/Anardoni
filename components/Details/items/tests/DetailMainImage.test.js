import React from 'react';
import { shallow } from 'enzyme';
import DetailMainImage from "../DetailMainImage";
describe("DetailMainImage component" , ()=>{
    it("it should render a div",()=>{
        const container = shallow(<DetailMainImage/>)
        expect(container).toMatchSnapshot();
    })
});