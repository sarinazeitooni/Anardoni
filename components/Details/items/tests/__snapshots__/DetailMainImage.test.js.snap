// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailMainImage component it should render a div 1`] = `ShallowWrapper {}`;
